.changeset/tender-ads-learn.md

```diff
@@ -0,0 +1,164 @@
+---
+"@salt-ds/lab": patch
+---
+
+SteppedTracker, Step and useStepReducer
```

Comment:
Needs to form a sentence for changelog, "Added", "Updated", etc.

There was an existing version of SteppedTracker live, so need to write a bit of before/after change for the user to migrate to the new version.

This file is probably too long for a changeset, hopefully most of these information is in the doc. If so, use something like "see more [link to doc]"